toffy/rosetta.py

```diff
@@ -491,3 +493,111 @@ def create_rosetta_matrices(default_matrix, save_dir, multipliers, masses=None):
                 mult_matrix.iloc[j, :] = comp_matrix.iloc[j, :] * i
         base_name = os.path.basename(default_matrix).split('.csv')[0]
         mult_matrix.to_csv(os.path.join(save_dir, base_name + '_mult_%s.csv' % (str(i))))
+
+
+def copy_image_files(cohort_name, run_names, rosetta_testing_dir, extracted_imgs_dir,
+                     fovs_per_run=5):
+    """ Creates a new directory for rosetta testing and copies over a random subset of
+        previously extracted images
+    Args:
+        cohort_name (str): name for all combined runs
+        run_names (list): gives names of run folders to retrieve extracted images from
+        rosetta_testing_dir: directory where to create cohort rosetta testing folder
+        extracted_imgs_dir: directory containing images from each run
+        fovs_per_run: number of fovs from each run to use for testing, default 5
```

Comment:
Should add a description in the notebook right before this is called telling users that 10-20 FOV is a good # to test on, and to adjust this parameter accordingly